src/taskEvents/script.js

```diff
@@ -213,6 +223,24 @@ function openAddSkillModal() {
   containerDiv.appendChild(skillCategoryDiv);
   containerDiv.appendChild(skillLevelDiv);
   containerDiv.appendChild(submitBtn);
+  
+  submitBtn.addEventListener('click',async () => {
+    const skillToAdd =skills?.find(skill => skill.name === skillCategorySelect.value)
+    const levelToAdd = level?.find(lvl => lvl.name === skillLevelSelect.value)
+    
+    submitBtn.innerHTML = `<div class="dot-flashing-loader"></div>`
+    submitBtn.classList.add("disabled")
+    await addSkillToUser(skillToAdd, levelToAdd, userid)
```

Comment:
We should check the response if its successful before proceeding

Reply:
done